### ISSUE TITLE
Explicitly referring to DRAGEN software usage

### ISSUE THREAD

**Issue by ChrOertlin** [Clinical-Genomics/cg#2030]
## Description

In some instances we are using DRAGEN as a constant in the CG codebase. These include constants that should point to a more specific software that is run on dragen. One such example is the BclConverter class. Here we have the options "bcl2fastq" and "dragen". However, dragen runs the "bclconvert" software, which is the more appropriate constant to use.

This issue is also carried over in some models that carry "Dragen" in the name but likely should point to a more specific software.

Being more explicit might help us identify potential issues faster.

One examlpe can be found in this PR:
https://github.com/Clinical-Genomics/cg/pull/2019

## Suggested solution

Find the usages of Dragen and if applicable be explicit of the underlying software that is used when we parse information. 

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by henrikstranneheim** [comment]
Perhaps let dragen be the key and the value "the software" used on the dragen. If there is merit keeping it around

**Comment by islean** [comment]
My current suggestion is that at least in the mentioned bcl converter class, the naming could be similar to

<img width="498" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/714c76ce-3ba2-426e-a66e-1a9d16a7d741">

where the only difference to previously is "DRAGEN: str = "dragen"" -> "DRAGEN_BCLCONVERT: str = "dragen_bcl_convert""

**Comment by ChrOertlin** [comment]
On hold for now
